pages/docs/user-docs/docs-recipes.md

```diff
@@ -2,60 +2,67 @@
-2. **Sections**: The rest of the definition is comprised of sections, or blobs of data. Each section is defined by a `%` character followed by the name of the particular section. All sections are optional.
+2. **Sections**: The rest of the definition is comprised of sections, sometimes called scriptlets or blobs of data. Each section is defined by a `%` character followed by the name of the particular section. All sections are optional.
 
+Please see the [examples](https://github.com/singularityware/singularity/tree/master/examples) directory in the [Singularity source code](https://github.com/singularityware/singularity) for some ideas on how to get started. 
 
 ### Header
-The header is at the top of the file, and tells Singularity the base to use, and which one. Specifically:
+The header is at the top of the file, and tells Singularity the base Operating System that it should use to build the container.  It is composed of several keywords. Specifically:
 
- - **Bootstrap**: references the kind of base you want to use (e.g., docker, debootstrap, shub). For example, a shub bootstrap will pull containers for shub as bases. A Docker bootstrap will pull docker layers to start your image. For a full list see <a href="/build">build</a>
- - **From**: is the named container (shub) or reference to layers (Docker) that you want to use (e.g., vsoch/hello-world)
+ - `Bootstrap:` references the kind of base you want to use (e.g., docker, debootstrap, shub). For example, a shub bootstrap will pull containers for shub as bases. A Docker bootstrap will pull docker layers to start your image. For a full list see <a href="/build">build</a>
+ - `From:` is the named container (shub) or reference to layers (Docker) that you want to use (e.g., vsoch/hello-world)
 
+Depending on the value assigned to `Bootstrap:`, other keywords may also be valid in the header.
 
-For example, a very minimal Docker build might look like this:
+For example, a very minimal Singularity Hub build might look like this:
 
-```bash
-Bootstrap: docker
-From: ubuntu:latest
+```
+Bootstrap: shub
+From: singularityhub:ubuntu
 ```
 
-a build that uses a mirror to install Centos-7 might look like this:
-
+A build that uses a mirror to install Centos-7 might look like this:
 
-```bash
-BootStrap: yum
+```
+Bootstrap: yum
 OSVersion: 7
 MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/$basearch/
 Include: yum
 ```
 
-Each different kind of build base comes with particular details during build time, and you can read about them at the following links:
+Each build base requires particular details during build time.  You can read about them and see examples at the following links:
 
- - [yum base](/build-yum)
- - [debootstrap base](/build-debootstrap)
- - [arch linux base](/build-arch)
- - [yum base](/build-docker)
- - [local image base](/build-localimage)
+ - [shub](/build-docker) (images hosted on Singularity Hub)
```

Comment:
this url doesn't exist I don't think...

Reply:
(note both are for docker)

Reply:
Most of these urls don't exist.  And their not going to until somebody can get a chance to write them lol.  This is a big job and is not going to get finished in a single PR.